### PR TITLE
Update logrotate and postrm

### PR DESCRIPTION
**Pull request by lucasmoura**

### CODE REVIEW

**Review by athos-ribeiro — APPROVED** [review]

**Review by sergiodj — APPROVED** [review]

**Review by blackboxsw — APPROVED** [review]
Confirmed behavior on upgrade. we're seeing all the logs we care about with the right perms and no errors from apt during install.
```
-rw------- 1 root root 10976 Oct  1 03:39 /var/log/ubuntu-advantage.log.1
-rw------- 1 root root  3628 Oct  5 21:06 /var/log/ubuntu-advantage.log
-rw------- 1 root root     0 Oct  5 21:06 /var/log/ubuntu-advantage-license-check.log
-rw------- 1 root root   157 Oct  5 21:06 /var/log/ubuntu-advantage-timer.log
```

Log rotate functions fine with the su root root inside the scope of the config.

File: features/attached_commands.feature [hunk 1]
```diff
@@ -31,13 +31,16 @@ Feature: Command behaviour when attached to an UA subscription
...
+        /var/log/ubuntu-advantage-license-check.log
         /var/log/ubuntu-advantage.log
         /var/log/ubuntu-advantage-timer.log
         """
         When I run `logrotate --force /etc/logrotate.d/ubuntu-advantage-tools` with sudo
         And I run `sh -c "ls /var/log/ubuntu-advantage* | sort -d"` as non-root
+        # We don't rotate empty log files. That's why the license-check logs are not rotated
         Then stdout matches regexp:
         """
+        /var/log/ubuntu-advantage-license-check.log
```
Comment (renanrodrigo) [inline features/attached_commands.feature:43]: Should not we have a `.1` here as well?

Comment (orndorffgrant) [reply features/attached_commands.feature:43]: There is a comment above.
```
# We don't rotate empty log files. That's why the license-check logs are not rotated
```

Comment (renanrodrigo) [reply features/attached_commands.feature:43]: :man_facepalming: 
I'm blind or something

**Pull request merged**